students/BrianNisonger/session9/test_mailroom.py

```diff
@@ -0,0 +1,181 @@
+from mailroom import Donor
+from mailroom import Donors
+import statistics
+
+
+def test_make_donor():
+    donor = Donor("Fred Flinstone")
+    assert donor.name == "Fred Flinstone"
+
+
+def test_make_donor_with_donation():
+    donor = Donor("Fred Flinstone", [100, 200, 300, 400])
+    assert donor.name == "Fred Flinstone"
+    assert donor.donation == [100, 200, 300, 400]
+
+
+def test_add_donation():
+    donor = Donor("Fred Flinstone")
+    donor.add_donation(500)
+    assert donor.donation == [
+        500,
+    ]
+
+
+def test_append_donation():
+    donor = Donor("Fred Flinstone")
+    donor.add_donation(500, 365.00)
+    assert donor.donation == [500, 365.00]
+    donor.add_donation(1007)
+    assert donor.donation == [500, 365.00, 1007]
```

Comment:
these tests are good, but do test the implementation a bit too much -- that is, that the Donor object stores the donations in order in a list -- that may not be true in the future.